packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -234,7 +234,8 @@ const ContextualPopupDecorator = hoc(defaultConfig, (config, Wrapped) => {
 				arrowPosition: {top: 0, left: 0},
 				containerPosition: {top: 0, left: 0},
 				containerId: Spotlight.add(this.props.popupSpotlightId),
-				activator: null
+				activator: null,
+				shouldSpotActivator: true
```

Comment:
It doesn't have to be a `state`, does it? If `shouldSpotActivator` is being assigned when `this.state.activator` gets reset, I don't see the point of making it a `state`.

Reply:
I try to avoid members whenever possible. They run the risk of becoming out of sync with state since state updates are batched up.